platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh

```diff
@@ -26,6 +26,18 @@ if [[ "$1" == "init" ]]; then
     modprobe nvram
     systemctl start s6100-reboot-cause.service
 
+    # Disable Xilinx/pericom chipset
+    bios_version=$(dmidecode -s system-version | cut -d "-" -f2)
+
+    if [[ $bios_version -gt 5 ]] ; then
```

Comment:
why do we need to check bios version?

Why not check if 08:00:0 exists or not, it exists, then remove?

Reply:
Sure Guhoan. sounds good..